deployment/distribution/build.gradle.kts

```diff
@@ -68,3 +72,18 @@ tasks {
         finalizedBy("installDist")
     }
 }
+
+publishing {
```

Comment:
✏️ I recommend looking at this [publish block from usng2](https://github.com/codice/usng2/blob/fff3d7a625a0e1e441b5507a9d5df870b39fd156/usng2-jvm/build.gradle.kts#L43-L79) which adds significant data to the generated pom (name, description, license, scm, etc.) and includes a test to discriminate between the snapshot and release repositories.

It also (assuming I've done it correctly; it needs to be tested) depends on the local maven settings for those repository urls which will be necessary so as not to require credentials in the build file itself to publish.